src/ClientContextInterface.php

```diff
@@ -114,6 +114,7 @@
  * @method $this zdiffstore(string $destination, array $keys)
  * @method $this zincrby($key, $increment, $member)
  * @method $this zinterstore(string $destination, array $keys, int[] $weights = [], string $aggregate = 'sum')
+ * @method $this zinter(array $keys, int[] $weights = [], string $aggregate = 'sum', bool $withScores = false)
```

Comment:
Should the keys and weights be a single array?

Reply:
The same as ZUNION. Keys should be array because of numkeys calculation and weights as array just easier to maintain